react/AvailabilityMessage.tsx

```diff
@@ -39,6 +40,12 @@ const AvailabilityMessage: StorefrontFunctionComponent<Props> = ({
     onRemove,
   } = useItemContext()
 
+  const { loading } = useOrderForm()
```

Comment:
As we talked, `product-list` should not depend on our orchestrators. Especially now that the Store Framework team is going to use our components. It's better to receive `loading` as prop and pass it down.